velox/exec/ExchangeClient.h

```diff
@@ -92,6 +92,8 @@ class ExchangeClient {
 
   void request(const RequestSpec& requestSpec);
 
+  // Handy for ad-hoc logging.
+  const std::string taskId_;
```

Comment:
Please inclue it in toJsonString. We have debugging endpoints which can print this, and can be useful to debug exchange bugs

Reply:
Sure. Done.